Demo/src/androidTest/java/com/braintreepayments/demo/test/utilities/CardNumber.java

```diff
@@ -20,6 +20,9 @@
     public static final String THREE_D_SECURE_MPI_LOOKUP_ERROR = "4000000000000085";
     public static final String THREE_D_SECURE_MPI_SERVICE_ERROR = "4000000000000093";
 
+    public static final String THREE_D_SECURE_2_FRICTIONLESS_VERIFICATON = "4000000000001000";
+    public static final String THREE_D_SECURE_2_CHALLENGE_VERIFICATON = "4000000000001091";
```

Comment:
both of these constants are Visa cards... is it worth changing one out for a test MC?

Reply:
These should be fine as we're testing the 3DS response from cardinal which should not be card brand dependent.